gamecenter/api/helpers/get.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+"""API helpers for GET requests."""
+import iso8601
+from flask import request
+from datetime import datetime as dt
+from sqlalchemy import and_
+
+from gamecenter.api.models import Score
+
+DEFAULTS = {
+    'page_size': 5,
+    'offset': 1,
+    'sort': 'descending',
+    'start_date': dt.fromtimestamp(0),
+    'end_date': dt.utcnow()
+    }
+
+
+def construct_and_(args):
+    """Construct the 'and_' clause for a sqlalchemy query"""
```

Comment:
each of these returns has the date filter in it, is there some way to like chain `and_`'s so that we can factor out the date filter to make this function a bit cleaner?